src/main/java/org/fundacionjala/pivotal/objects/CreateProject.java

```diff
@@ -0,0 +1,132 @@
+package org.fundacionjala.pivotal.objects;
+
+import org.fundacionjala.pivotal.common.Base;
+import org.fundacionjala.pivotal.common.CommonMethods;
+import org.fundacionjala.pivotal.common.commonMeth;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for create a Project.
+ */
+public class CreateProject extends Base {
+
+    @FindBy(how = How.ID, using = "create-project-button")
+    private WebElement buttonCreateProject;
+
+    @FindBy(how = How.NAME, using = "project_name")
+    private WebElement txtProjectName;
+
+    @FindBy(how = How.XPATH, using = "//div[@class='tc-account-selector__header']")
+    private WebElement txtSelectorAcccountCheck;
+
+    ////td[text()='UserID']            @FindBy(how = How.CSS, using = "#view42 span.raw_context_name")
+    @FindBy(how = How.CLASS_NAME, using = "raw_context_name")
+    private WebElement txtVerifyName;
+
+    @FindBy(how = How.CSS, using = "#view42 img.headerLogo__image")
+    private WebElement goToDashBoard;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"modal_area\"]/div/div[2]/div/form/footer/button[2]")
+    private WebElement buttonCreate;
+
+    @FindBy(how = How.CSS, using = "#modal_area  span.tc-account-selector__create-account-text")
+    private WebElement buttonCreateAccount;
+
+    @FindBy(how = How.CSS, using = "#modal_area  input.tc-account-creator__name")
+    private WebElement txtAccountName;
+
+    /**
+     * Set go to DashBoard.
+     */
+    public void setGoToDashBoard() {
+        driverWait.until(ExpectedConditions.visibilityOf(goToDashBoard));
+        goToDashBoard.click();
+    }
+
+    /**
+     * Method for verifyName.
+     * @return txtVerifyName.
+     */
+    public String verifyName() {
+        CommonMethods.waitWebElement(txtVerifyName);
+        return txtVerifyName.getText();
+    }
+
+    /**
+     * Method for set Privacy.
+     * @param privacy String.
+     */
+    public void setPrivacy(String privacy) {
+        if (privacy.equalsIgnoreCase("private")) {
+            driver.findElement(By.cssSelector("div#modal_area label:nth-child(2) > input[name=\"project_type\"]")).click();
+        }
+        if (privacy.equalsIgnoreCase("public")) {
+            driver.findElement(By.cssSelector("div#modal_area label:nth-child(3) > input[name=\"project_type\"]")).click();
+        }
+    }
+
+    /**
+     * Method for set Select an Account Specific.
+     * @param account String.
+     */
+    public void setTxtSelectorAcccountSpecific(String account) {
+        this.setTxtSelectorAcccountCheck();
+        //CommonMethods.clickWebElement(driver.findElement(By.xpath("//div[@class='tc-account-selector__option-account-name' and text()='" + account + "']")));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)